components/user/UserEddiehubData.js

```diff
@@ -0,0 +1,29 @@
+import UserEddiehubDataItem from "./UserEddiehubDataItem";
+
+export default function UserEddiehubData({ data }) {
```

Comment:
What about having the `EddieHub` components in a sub directory `components/EddieHub/Data.js`?

Reply:
Sure, I'm not too opinionated about the file structure. I'm happy to move any anywhere else 👍